drizzlepac/helpers/utils.py

```diff
@@ -0,0 +1,245 @@
+import os
+import re
+import requests
+from astropy.io import fits
+
+__all__ = ['cmp_fitshdr', 'cmp_gen_hdrkeywords',
+           'word_precision_check', 'abspath',
+           'download', 'check_url']
+
+RE_URL = re.compile('\w+://\S+')
+
+default_compare = dict(
+    ignore_keywords=['DATE', 'CAL_VER', 'CAL_VCS', 'CRDS_VER', 'CRDS_CTX'],
+    keys=['primary', 'sci', 'dq'],
+    rtol=0.000001,
+)
+
+
+def cmp_fitshdr(left, right, **kwargs):
```

Comment:
Why not just use http://astropy.readthedocs.io/en/latest/io/fits/api/diff.html#astropy.io.fits.HeaderDiff ?

`fits.HDUList([a[kw] for kw in keys])` should not be necessary (at least in theory).

Reply:
Those `cmp_*` functions were written to encapsulate the ~10 lines of code used per test in `jwst.git`. If there's a cleaner way to do any of this... we should. I was only basing those functions off what I was seeing at the time.